relayer/naive-strategy.go

```diff
@@ -714,7 +717,7 @@ func relayPacketsFromResultTx(src, dst *Chain, res *ctypes.ResultTx) ([]relayPac
 
 			switch {
 			// If the packet has a timeout height, and it has been reached, return a timeout packet
-			case rp.timeout != 0 && block.GetHeight().GetRevisionHeight() >= rp.timeout:
+			case rp.timeout.RevisionHeight != 0 && block.GetHeight().GetRevisionHeight() >= rp.timeout.RevisionHeight:
```

Comment:
```suggestion
			case !rp.timeout.IsZero() != 0 && block.GetHeight().GTE(rp.timeout):
```

we should match this check https://github.com/cosmos/ibc-go/blob/372eaa42a9082bcbeaca344fc41151ab95f8b5cd/modules/core/04-channel/keeper/packet.go#L209